package.json

```diff
@@ -30,6 +30,7 @@
     },
     "dependencies": {
         "@angular/animations": "^7.2.7",
+        "@angular/cdk": "^7.3.7",
```

Comment:
I think we wanted to include the cdk as a dependency of the actual library. I still think we should run this by the guys in the fundamentals meeting.